### ISSUE TITLE
Local Call Log Deserialization

### ISSUE THREAD

**Issue by mrtpain** [stratisproject/StratisFullNode#661]
As an API user, I would like to have Local Call logs deserialized before the request is returned from `SmartContracts/local-call`. Currently the `internalTransfers` do not output the address values and log `topics` and `data` are not returning deserialized values as they do at the `SmartContracts/receipt` endpoint. 

Using master branch 60cff14fbcd050e02bad0c4e5dbbefb26ba41018

**Current Response**
```
{
    "internalTransfers": [
        {
            "from": {
                "size": 20
            },
            "to": {
                "size": 20
            },
            "value": 0
        }
    ],
    "gasConsumed": {
        "value": 33810
    },
    "revert": false,
    "errorMessage": null,
    "return": null,
    "logs": [
        {
            "address": {
                "size": 20
            },
            "topics": [
                "VHJhbnNmZXJMb2c=",
                "FNys6Fs53wRCRQyDlaFu5ODb+I8=",
                "ic5TcYHmoKaTwsCEUOOsw9kqxfg="
            ],
            "data": "+EuUFNys6Fs53wRCRQyDlaFu5ODb+I+Uic5TcYHmoKaTwsCEUOOsw9kqxfigECcAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA="
        },
        {
            "address": {
                "size": 20
            },
            "topics": [
                "U3RhcnRNaW5pbmdMb2c=",
                "FNys6Fs53wRCRQyDlaFu5ODb+I8="
            ],
            "data": "+HiUFNys6Fs53wRCRQyDlaFu5ODb+I+gECcAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACgUOKjcuEVjQUAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACgUOJU5qwteAMAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA="
        }
    ]
}
```

**Expected Response**
```
{
    "internalTransfers": [
        {
            "from": "PAVV2c9Muk9Eu4wi8Fqdmm55ffzhAFPffV",
            "to": "PM9pu2ppUaRU1WMsUnq5en1b4ruwLqqEpX",
            "value": 0
        }
    ],
    "gasConsumed": {
        "value": 33810
    },
    "revert": false,
    "errorMessage": null,
    "return": null,
    "logs": [
        {
            "address": "PX2J4s4UHLfwZbDRJSvPoskKD25xQBHWYi",
            "topics": [
                "5472616E736665724C6F67",
                "14DCACE85B39DF0442450C8395A16EE4E0DBF88F",
                "89CE537181E6A0A693C2C08450E3ACC3D92AC5F8"
            ],
            "data": "F84B9414DCACE85B39DF0442450C8395A16EE4E0DBF88F9489CE537181E6A0A693C2C08450E3ACC3D92AC5F8A0002D310100000000000000000000000000000000000000000000000000000000",
            "log": {
                "event": "TransferLog",
                "from": "PAVV2c9Muk9Eu4wi8Fqdmm55ffzhAFPffV",
                "to": "PM9pu2ppUaRU1WMsUnq5en1b4ruwLqqEpX",
                "amount": "20000000"
            }
        },
        {
            "address": "PM9pu2ppUaRU1WMsUnq5en1b4ruwLqqEpX",
            "topics": [
                "53746172744D696E696E674C6F67",
                "14DCACE85B39DF0442450C8395A16EE4E0DBF88F"
            ],
            "data": "F8789414DCACE85B39DF0442450C8395A16EE4E0DBF88FA0002D310100000000000000000000000000000000000000000000000000000000A040BBA372E1158D05000000000000000000000000000000000000000000000000A040BB54E6AC2D7803000000000000000000000000000000000000000000000000",
            "log": {
                "event": "StartMiningLog",
                "miner": "PAVV2c9Muk9Eu4wi8Fqdmm55ffzhAFPffV",
                "amount": "20000000",
                "totalSupply": "399999999941000000",
                "minerBalance": "249999999941000000"
            }
        }
    ]
}
```

**Comment by fassadlr** [comment]
I believe this has been merged to 1.1.1.0 as well as the opdex branch. Can we close this?

https://github.com/stratisproject/StratisFullNode/pull/805

**Comment by mrtpain** [comment]
> I believe this has been merged to 1.1.1.0 as well as the opdex branch. Can we close this?
> 
> #805

Yep, closing this. The changes we had in the opdex branch resolved the issue, with that merged to 1.1.1.0, this is done.
